lib/dotcom_web/components/system_status/widget.ex

```diff
@@ -0,0 +1,150 @@
+defmodule DotcomWeb.Components.SystemStatus.Widget do
+  @moduledoc """
+  A component that renders the given `@statuses` in a table.
+  """
+
+  use DotcomWeb, :component
+
+  import DotcomWeb.Components.RoutePills
+  import DotcomWeb.Components.SystemStatus.StatusLabel
+
+  @route_ids ["Red", "Orange", "Green", "Blue"]
+
+  def system_status_widget(assigns) do
```

Comment:
Please add the requisite `attr` here, looks like just `:routes_with_statuses`?

Reply:
Yep!

Also I don't like the name `@routes_with_statuses`. What about `@subway_status`?